service/federations/certificates/requests.go

```diff
@@ -0,0 +1,200 @@
+package certificates
```

Comment:
These certificates are used for SAML-based federations only, so it's better to move this package inside the `saml` directory.

Reply:
Moved it